code/game/objects/items/shuttle_upgrades.dm

```diff
@@ -0,0 +1,55 @@
+/obj/item/shuttle_route_optimisation
+	name = "Route Optimisation Upgrade"
+	desc = "Used on a custom shuttle control console to calculate more efficient routes."
+	icon = 'icons/obj/module.dmi'
+	icon_state = "shuttledisk"
+	force = 0
+	throwforce = 8
+	throw_speed = 3
+	throw_range = 5
+	density = FALSE
+	anchored = FALSE
+	item_flags = NOBLUDGEON
+	var/upgrade_amount = 0.8
+
+/obj/item/shuttle_route_optimisation/hyperlane
+	name = "Bluespace Hyperlane Calculator"
+	desc = "Used on a custom shuttle control console to allow for the following of bluespace hyperlanes, increasing the efficiency of the shuttle."
+	icon = 'icons/obj/module.dmi'
+	icon_state = "shuttledisk_better"
+	force = 0
+	throwforce = 8
+	throw_speed = 3
+	throw_range = 5
+	density = FALSE
+	anchored = FALSE
+	item_flags = NOBLUDGEON
```

Comment:
don't need to re-define all this for this if it's parent has exactly the same stats